### PR TITLE
dev, v3.0, v3.1: add GC I/O limit

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 
add GC I/O limit

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/2015

### Which version does your change affect? 
dev, v3.0, v3.1

### CODE REVIEW

File: v3.0/reference/garbage-collection/configuration.md [hunk 1]
```diff
@@ -103,4 +103,18 @@ Since TiDB 3.0, some configuration options have changed with support for the dis
 - Serial: requests are sent from TiDB Region by Region.
 - Concurrent: requests are sent to each Region concurrently based on the number of threads specified in the `tikv_gc_concurrency`.
 - Auto-concurrent: requests are sent to each Region concurrently with the number of TiKV nodes as concurrency value.
-- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
\ No newline at end of file
+- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
+
+## GC I/O limit
+
+Starting from v3.0.6, TiKV supports GC I/O limit. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```
Comment (lilin90) [inline v3.0/reference/garbage-collection/configuration.md]:
```suggestion
TiKV supports GC I/O limit <span class="version-mark">New in v3.0.6</span>. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```

File: dev/reference/garbage-collection/configuration.md [hunk 2]
```diff
@@ -103,4 +103,18 @@ Since TiDB 3.0, some configuration options have changed with support for the dis
 - Serial: requests are sent from TiDB Region by Region.
 - Concurrent: requests are sent to each Region concurrently based on the number of threads specified in the `tikv_gc_concurrency`.
 - Auto-concurrent: requests are sent to each Region concurrently with the number of TiKV nodes as concurrency value.
-- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
\ No newline at end of file
+- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
+
+## GC I/O limit
+
+Starting from v3.0.6, TiKV supports GC I/O limit. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```
Comment (lilin90) [inline dev/reference/garbage-collection/configuration.md]:
```suggestion
TiKV supports the GC I/O limit. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```

File: v3.1/reference/garbage-collection/configuration.md [hunk 3]
```diff
@@ -103,4 +103,18 @@ Since TiDB 3.0, some configuration options have changed with support for the dis
 - Serial: requests are sent from TiDB Region by Region.
 - Concurrent: requests are sent to each Region concurrently based on the number of threads specified in the `tikv_gc_concurrency`.
 - Auto-concurrent: requests are sent to each Region concurrently with the number of TiKV nodes as concurrency value.
-- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
\ No newline at end of file
+- Distributed: no need for TiDB to send requests to TiKV to trigger GC because each TiKV handles GC on its own.
+
+## GC I/O limit
+
+Starting from v3.0.6, TiKV supports GC I/O limit. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```
Comment (lilin90) [inline v3.1/reference/garbage-collection/configuration.md]:
```suggestion
TiKV supports the GC I/O limit. You can configure `gc.max-write-bytes-per-sec` to limit writes of a GC worker per second, and thus to reduce the impact on normal requests.
```

**Comment by youjiali1995** [comment]
> GC I/O limit is supported in `dev` and `v3.1`. Right? @youjiali1995

dev, v3.1 and v3.0

**Comment by lilin90** [comment]
@youjiali1995 Got it. Thanks! Then it's unnecessary to let users know we support it starting from v3.0.6 in dev and v3.1.

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

**Pull request merged**